#! /bin/sh /usr/share/dpatch/dpatch-run
## 13_mplayer.desktop_polish_translation.dpatch by Kevin Goodsell <kevin-opensource@omegacrash.net>
##
## DP: Polish translation from LP: #93382.

@DPATCH@
diff -urNad mplayer-trunk~/etc/mplayer.desktop mplayer-trunk/etc/mplayer.desktop
--- mplayer-trunk~/etc/mplayer.desktop	2009-07-14 10:22:09.000000000 -0700
+++ mplayer-trunk/etc/mplayer.desktop	2009-07-14 10:23:32.000000000 -0700
@@ -2,13 +2,16 @@
 Type=Application
 Name=MPlayer Media Player
 Name[ca]=Reproductor multimèdia MPlayer
+Name[pl]=Odtwarzacz filmów MPlayer
 GenericName=Multimedia player
+GenericName[pl]=Odtwarzacz filmów
 Comment=Play movies and songs
 Comment[ca]=Reproduïu vídeos i cançons
 Comment[de]=Filme und Musik abspielen
 Comment[es]=Reproduzca vídeos y canciones
 Comment[fr]=Lecteur multimédia
 Comment[it]=Lettore multimediale
+Comment[pl]=Odtwarzacz multimediów
 Comment[zh]=多媒体播放器
 Icon=mplayer
 TryExec=gmplayer
